examples/src/main/java/nl/knaw/dans/lib/dataverse/example/DatasetPublish.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2021 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.lib.dataverse.example;
+
+import nl.knaw.dans.lib.dataverse.DataverseResponse;
+import nl.knaw.dans.lib.dataverse.ExampleBase;
+import nl.knaw.dans.lib.dataverse.model.file.FileMeta;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class DatasetPublish extends ExampleBase {
+
+    private static final Logger log = LoggerFactory.getLogger(DatasetPublish.class);
+
+    public static void main(String[] args) throws Exception {
+        String persistentId = args[0];
+        log.info("--- BEGIN JSON OBJECT ---");
+        DataverseResponse<List<FileMeta>> r = client.dataset(persistentId).publish();
+        log.info("--- END JSON OBJECT ---");
+        log.info("Response message: {}", r.getEnvelopeAsJson().toPrettyString());
```

Comment:
There should be some lines that demontrate how you can access the JSON object returned as a Java object.